bin/reformat_rhea_chebi_mapping.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright 2024 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import gzip
+import re
+from pathlib import Path
+
+
+def convert_to_tsv(gz_file_path: Path, output_file_name: Path) -> Path:
+    with gzip.open(gz_file_path, "rt") as file_in, open(output_file_name, "w") as file_out:
+        input_text = file_in.read()
+        pattern = re.compile(
+            r"ENTRY\s+(\S+)\nDEFINITION\s+(.+?)\nEQUATION\s+(.+?)(?:\nENZYME\s+(.+?))?\n///",
+            re.DOTALL
+        )
+        matches = pattern.findall(input_text)
+        file_out.write("ENTRY\tDEFINITION\tEQUATION\tENZYME\n")
+        for match in matches:
+            entry, definition, equation, enzyme = match
+            enzyme = " ".join(enzyme.strip().split()) if enzyme else ""
+            file_out.write(f"{entry}\t{definition}\t{equation}\t{enzyme}\n")
```

Comment:
Please use the csv module, something like this. The csv module will handle edge cases for you (like having a space in of the entries for example).

```suggestion
          writer = csv.writer(file_out, delimiter='\t')
          writer.writerow(['ENTRY', 'DEFINITION', 'EQUATION', 'ENZYME'])
          for match in matches:
              entry, definition, equation, enzyme = match
              enzyme = " ".join(enzyme.strip().split()) if enzyme else ""
              writer.writerow([entry, definition, equation, enzyme])
```